src/test/integration/integration_tests.py

```diff
@@ -0,0 +1,276 @@
+                self.assertEqual(id, attributes["organizationCode"])
+                self.assertEqual(bc, attributes["businessCenter"])
+
+    def test_departments_bad_requests(self):
+        self.assertEqual(self.__departments_request(None).status_code, 400)
+        self.assertEqual(self.__departments_request("Bad bc").status_code, 400)
+
+    def __positions_request(self, business_center, position_type):
+        params = {'businessCenter': business_center,
+                  'type': position_type}
+        return self.__make_request(self.positions_url, params)
+
+    def test_positions(self):
+        for bc in self.business_centers:
+            logging.debug("Testing business center: {}".format(bc))
+
+            departments = self.__departments_request(bc).json()
+            department_codes = set([department["id"]
+                                   for department in departments["data"]])
+
+            logging.debug("Department codes: {}".format(department_codes))
+
+            request = self.__positions_request(bc, self.position_type)
+            request_elapsed_seconds = request.elapsed.total_seconds()
+            logging.debug("Request took {} second(s)"
+                          .format(request_elapsed_seconds))
+
+            self.assertEqual(request.status_code, 200)
+            self.assertLess(request_elapsed_seconds, 1)
+
+            positions = request.json()["data"]
+            self.assertGreaterEqual(len(positions), 1)
+
+            for position in positions:
+                id = position["id"]
+                logging.debug("Testing position ID: {}".format(id))
+                attributes = position["attributes"]
+
+                self.assertEqual(position["type"], "position")
+                self.assertIsNotNone(id)
+                self.assertIsNotNone(attributes["title"])
+                self.assertIsNotNone(attributes["class"])
+                self.assertEqual(attributes["businessCenter"], bc)
+                self.assertTrue(attributes["organizationCode"]
+                                in department_codes)
+
+                low_salary = attributes["lowSalaryPoint"]
+                high_salary = attributes["highSalaryPoint"]
+                self.assertIsNotNone(low_salary)
+                self.assertIsNotNone(high_salary)
+                self.assertGreater(high_salary, low_salary)
+
+    def test_position_type_case(self):
+        request = self.__positions_request(self.business_centers[0],
+                                           self.position_type.upper())
+        self.assertEqual(request.status_code, 200)
+
+    def test_positions_bad_requests(self):
+        self.assertEqual(self.__positions_request(
+                         None, self.position_type).status_code, 400)
+        self.assertEqual(self.__positions_request(
+                         "Bad bc", self.position_type).status_code, 400)
+        self.assertEqual(self.__positions_request(
```

Comment:
Could you also add one test case which `type` parameter is `None` like you did at [line 148](https://github.com/osu-mist/hr-api/blob/d0c624a3bf4176d0fdcbb266eca49024c354f4d5/src/test/integration/integration_tests.py#L148)?

Reply:
Done.